### PR TITLE
[Issue #36] Moves Vagrant guide from Wiki to docs

### PR DESCRIPTION
**Pull request by amoose**
### Addresses #36 

First draft of a Contributing guide
Adds license to README with link to LICENSE.md
Adds link to contributing guide to README

### CODE REVIEW

**Comment by amoose** [comment]
@alex @llimllib @harrisj please peek at a first draft of a [contributing](https://github.com/department-of-veterans-affairs/connect_vbms/blob/feature/vagrant_docs/docs/contributing.rst) guide. I mention git-flow although we're not technically using it.

**Comment by llimllib** [comment]
New contributors won't be able to push to our repo unless given access; presumably we should tell them to fork the repo and push to their copy?

Do we really want to instruct people on how to do so rather than just point them to [this page](https://help.github.com/articles/using-pull-requests/) or something similar?

Should this commit be separate from the Vagrant commit, since it seems to be orthogonal to it?

**Comment by amoose** [comment]
I intentionally did not go in to detail on how to create a pull request. The imperative information is in how developers present their work for review; which branch to branch from and which to pull request to. The same applies to forking vs. cloning and pushing; it's standard GitHub practices and [guides are plentiful](https://help.github.com/). 

I also didn't break out the contributing guide since the developer-facing path is README > Contributing > Developing with Vagrant, however I am happy to do so.

**Comment by amoose** [comment]
I will separate these commits for each guide. :tv:

**Comment by harrisj** [comment]
These docs look good to me, but should we also link them into the index.rst for the gem?

File: docs/index.rst [hunk 1]
```diff
@@ -26,7 +31,7 @@ Alternatively, you can set the values for these fields as environment variables.
 
 .. _12-factor application: http://12factor.net/
 
-.. code-block:: zsh
```
Comment (amoose) [inline docs/index.rst]: This works on GitHub but not with other editors; `shell` seems more appropriate.

File: docs/index.rst [hunk 2]
```diff
@@ -60,15 +65,38 @@ Connect VBMS works by creating request objects, which are pure-data objects to
...
+Java Versions
+=============
+
+To build with a specific version of Java, use our `building java guide`_.
 
-Contents
---------
+----
 
-.. toctree::
```
Comment (amoose) [inline docs/index.rst]: This doesn't render on GitHub _or_ the other editors I tried. Went with `contents` for auto-generated TOC.

File: docs/index.rst [hunk 3]
```diff
@@ -60,17 +65,39 @@ Connect VBMS works by creating request objects, which are pure-data objects to
...
+Contributing Guide
+==================
+
+View our `contributing guide`_ for information on contributing to this gem. 
+
+Developing with Vagrant
+=======================
+
+View our `vagrant guide`_ for information on using our prebuilt Vagrant VM for development.
+
```
Comment (amoose) [inline docs/index.rst]: these URLs will be active once merged to master

**Comment by harrisj** [comment]
This has been merged

**Pull request closed without merging**